platform/src/main/java/org/stellar/anchor/platform/action/DoStellarRefundHandler.java

```diff
@@ -70,6 +70,15 @@ protected void validate(JdbcSepTransaction txn, DoStellarRefundRequest request)
             .build(),
         true,
         assetService);
+
+    if (!txn.getAmountInAsset().equals(request.getRefund().getAmount().getAsset())) {
+      throw new InvalidParamsException(
+          "refund.amount.asset does not match transaction amount_fee_asset");
```

Comment:
This messages seems to be incorrect.
Shouldn't it be "refund.amount.asset does not match transaction amount_in_asset"?